jobs/config.json

```diff
@@ -12636,6 +12636,16 @@
       "sig-openstack"
     ]
   },
+  "pull-cluster-api-all": {
```

Comment:
Please move this below "pull-cluster-api-build" as it makes more sense for the most basic scenario (running the build) to be the first item in this list.

Reply:
i believe the file is sorted :)

Reply:
Yeah, FWIW this file is going away soon, these will be inlined into the Prow config definitions of the jobs... This is mostly legacy from Jenkins + Prow but it works.

Reply:
Got it -- ignore all my comments about ordering of the items.

Reply:
we should un-ignore them once we're no longer auto-sorting cc @krzyzacy